odd-platform-ui/src/components/shared/AppGraph/AppGraphLink/AppGraphLink.tsx

```diff
@@ -67,12 +67,20 @@ const AppGraphLink: React.FC<AppGraphLinkProps> = ({
   const drawPath = () =>
     linkHorizontal()(crossLink ? crossLinkCoords : coords) || undefined;
 
+  const setMarkerStart = (): string => {
+    if (reverse && !crossLink) return 'url(#head)';
+    if (crossLink) return 'url(#crossHead)';
```

Comment:
No need to check crossLink twice:
```suggestion
    if (crossLink) return 'url(#crossHead)';
    if (reverse) return 'url(#head)';
    return '';
```

Reply:
In this component now we have a few different states: AppGraphLink, AppGraphLink reverse, AppGraphLink crossLink and AppGraphLink reverse crossLink, so i need this double check for AppGraphLink reverse state.